src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityThread.test.js

```diff
@@ -1,22 +1,68 @@
 // @flow
+
 import React from 'react';
 import { shallow, ShallowWrapper } from 'enzyme';
-
 import ActivityThread from '../ActivityThread.js';
+import ActivityThreadReplies from '../ActivityThreadReplies';
+import PlainButton from '../../../../../components/plain-button';
 
 describe('src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread', () => {
-    const getWrapper = (props = {}): ShallowWrapper => shallow(<ActivityThread {...props}>Test</ActivityThread>);
+    const defaultProps = {
+        replies: [
+            {
+                type: 'comment',
+                id: '2485934',
+                message: 'test reply',
+            },
+            {
+                type: 'comment',
+                id: '24835934',
+                message: 'test reply',
+            },
+        ],
+        total_reply_count: 2,
+        hasReplies: true,
+    };
 
-    test('should render children component wrapped in ActivityCard if hasReplies is true', () => {
-        const wrapper = getWrapper({ hasReplies: true });
+    const getWrapper = (props = {}): ShallowWrapper =>
```

Comment:
I believe we're favoring testing-library instead of enzyme, could these tests use that instead?

Reply:
Thanks! Rewritten to use testing-library 👍